custom_functions/fast_etl.py

```diff
@@ -232,6 +241,75 @@ def compare_source_dest_rows(source_cur,
                       f'Origem: {source_row_count} linhas. ' \
                       f'Destino: {destination_row_count} linhas')
 
+
+def get_hook_and_engine_by_provider(
+        provider: str,
+        conn_id: str) -> [DbApiHook, Engine]:
+    if provider == 'MSSQL':
+        hook = MsSqlHook(conn_id)
+        engine = get_mssql_odbc_engine(conn_id)
+    elif provider == 'PG':
+        hook = PostgresHook(conn_id)
+        engine = hook.get_sqlalchemy_engine()
+    elif provider == 'MYSQL':
+        hook = MySqlHook(conn_id)
+        engine = hook.get_sqlalchemy_engine()
+
+    return hook, engine
+
+
+def _convert_column(old_col: Column, db_provider: str) -> Column:
+    type_mapping = {
+        'NUMERIC': sa_types.Numeric(38, 13),
```

Comment:
Faria sentido colocar esses valores `38`, `13` como constantes no início do arquiivo?

Reply:
Acho que inicialmente poderia usar o datatype Float retornado pelo SqlAlchemy, em que a precision e scale são dinâmicas e permite os tamanhos máximos.

Reply:
Esse é o valor máximo aceito. Até onde pensei não tem como prever o tamanho perfeito, por isso utilizei o tamanho máximo. Essa solução pode ser evoluída. É um tradeoff que temos que aceitar, pelo menos enquanto não melhoramos a solução.

Reply:
É usado apenas uma vez. Então por mim tudo bem deixar aí. Se necessário, refatoramos depois.

Reply:
@washolive ao utilizar o tipo Float tive problemas para escrever dados com o pyodbc, a lib utilizada no FastETL. Para simular isso basta comentar essa linha 263 e rodar os testes automáticos.